server/text_generation_server/inference_engine/tgis_native.py

```diff
@@ -52,6 +53,11 @@ def __init__(
                 raise NotImplementedError(
                     f"Flash attention currently only supported by the following model types: {NONTP_FLASH_TYPES}"
                 )
+        elif PAGED_ATTENTION:
```

Comment:
I think right now we require both `PAGED_ATTENTION` *and* `FLASH_ATTENTION` to be set, so not sure if this should be `elif`.